readers/messages.go

```diff
@@ -20,7 +20,8 @@ var ErrNotFound = errors.New("entity not found")
 type MessageRepository interface {
 	// ReadAll skips given number of messages for given channel and returns next
 	// limited number of messages.
-	ReadAll(string, uint64, uint64, map[string]string) (MessagesPage, error)
+	ReadAllByChanID(string, uint64, uint64, map[string]string) (MessagesPage, error)
```

Comment:
I would not personally change the name of this function... We are always reading by `chanID` - just that we are sometimes also filtering by `thingID`, and somtimes not.

Reply:
@drasko Yes I agree, and I think by reverting its name we have fewer changes in this PR.